purepyvmware/datastores.py

```diff
@@ -0,0 +1,329 @@
+import time
+
+from lib import datastore_utils
+from pyVmomi import vim
+from pyVmomi import sms
+
+
+class Datastores(object):
+    """Maintenance of Pure Storage based datastores and associated array volumes.
+
+    For general maintenance of VMware datastores that are backed by Pure Storage FlashArrays.
+    """
+
+    def __init__(self, vsphere_content, flasharray):
+        """Setting up vSphere and FlashArray environment for use.
+
+        Args:
+            vsphere_content (vim.ServiceInstanceContent): All available content within the vSphere env.
+            flasharray (purestorage.FlashArray): Existing session to FlashArray.
+        """
+        self.flasharray = flasharray
+        self.pure_identifier = '24a937'
```

Comment:
Is this something which should be hidden? or is it public knowledge?

Reply:
This is public knowledge, so we can keep it as is. :-)

-jhop